dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -10,7 +10,7 @@ RUN apt-get update
 
 RUN apt-get install -f -y ifupdown arping libdbus-1-3 libdaemon0 libjansson4 libpython2.7 iproute2
 
-RUN apt-get install -f -y ndisc6 tcpdump python-scapy
+RUN apt-get install -f -y ndisc6 tcpdump python-scapy=2.4.0-2
```

Comment:
Can we install scapy through pip instead so we don't need include debian unstable repo?
for example use 2.4.2 version: 
RUN pip install scapy==2.4.2

Reply:
good idea.